EnglishTranslation-master/SKILL.tsv

```diff
@@ -4601,11 +4601,11 @@ SKILL_20150714_004600	Gun Mastery: Accuracy
 SKILL_20150714_004601	Increases your accuracy by 30 per attribute level when equipped with a Pistol-type weapon.
 SKILL_20150714_004602	Increases the damage dealt on an enemy with [Cure] by 1% per attribute level.
 SKILL_20150714_004603	Increases the damage dealt on an enemy with [Heal] by 1% per attribute level.
-SKILL_20150714_004604	Enemies hit with [Strike] has 2% chance per Attribute level to fall under Stun.
-SKILL_20150714_004605	When using [Heal], you will receive the same effects too at a 2% chance per attribute level.
+SKILL_20150714_004604	Enemies hit with a [Strike] type attack have a 2% chance per attribute level to become afflicted with [Stun].
+SKILL_20150714_004605	Adds the chance to automatically be healed at a 2% chance per attribute level when using [Heal].
```

Comment:
I'm not entirely sure how this attribute works but is the proc upon casting Heal, or does it happen when an ally is healed by a Heal tile?